todo_app/app.py

```diff
@@ -1,11 +1,33 @@
-from flask import Flask
-
+from flask import Flask, render_template, request
 from todo_app.flask_config import Config
+import requests
+import os
+from todo_app.data.item import Item
+import todo_app.data.trello_items as trello_items
+from todo_app.data.get_items import ViewModel
 
-app = Flask(__name__)
-app.config.from_object(Config())
-
-
-@app.route('/')
-def index():
-    return 'Hello World!'
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config())
+   
+    @app.route('/')
+    def index():
+        response_json = trello_items.get_trello_cards()
+        items = []
+        get_items = ViewModel(items)
+        for trello_list in response_json:
+            for card in trello_list['cards']:
+                item = Item.from_trello_card(card, trello_list)
+                items.append(item)
```

Comment:
This works but it's not super intuitive passing through the array and then populating it! Switching the order feels much more natural to me
```suggestion
        for trello_list in response_json:
            for card in trello_list['cards']:
                item = Item.from_trello_card(card, trello_list)
                items.append(item)
        get_items = ViewModel(items)
```